mythril/laser/ethereum/plugins/implementations/mutation_pruner.py

```diff
@@ -48,17 +53,21 @@ def staticcall_mutator_hook(global_state: GlobalState):
 
         @symbolic_vm.laser_hook("add_world_state")
         def world_state_filter_hook(global_state: GlobalState):
-            if And(
-                *global_state.world_state.constraints[:]
-                + [
-                    global_state.environment.callvalue
-                    > symbol_factory.BitVecVal(0, 256)
-                ]
-            ).is_false:
-                return
+
             if isinstance(
                 global_state.current_transaction, ContractCreationTransaction
             ):
                 return
+
+            try:
+                constraints = global_state.world_state.constraints + [
+                    global_state.environment.callvalue > 0
```

Comment:
I think we should use UGE here, because callvalue is an unsigned value

Reply:
UGT right? Done but it required some ugly type normalisation.